pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "epcsunspecdemo"
+version = "0.1.0.post46.dev0+2a57176"
+description = ""
+authors = ["Alex Anker <alex.anker@epcpower.com>"]
```

Comment:
Probably more appropriate to put a generic email address, if we include an `authors` entry at all. Perhaps `info at epcpower`?

Reply:
doesnt work without an author

Reply:
```suggestion
authors = ["EPC Power Corporation <info@epcpower.com>"]
```

Reply:
spoke about this, will leave in my name and what happens

Reply:
Good luck!